pages/api/module/post.ts

```diff
@@ -0,0 +1,24 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { Module } from "../models/module";
+
+const post = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    let postModule;
+    if (!req.body.title)
+        return res.status(400).send("Please provide a title for your module.");
+    else {
+        postModule = new Module({
+            title: req.body.title,
+            tool: req.body.tool,
+            slides: req.body.slides,
```

Comment:
For the optional fields, have you tested whether this works as it is currently -- since you are accessing req.body for specific keys that might not exist

Reply:
Yup! The only mandatory field is the title, all other fields default as expected.